pkg/bootstrapper/service.go

```diff
@@ -65,6 +65,33 @@ func (service *kubeletService) startKubeletService() error {
 	if err := service.obj.Start(); err != nil {
 		return err
 	}
+
+	if len(service.svcDependents) != 0 {
```

Comment:
Same trick mentioned before can be done here:
```suggestion
	if len(service.svcDependents) == 0 {
	    return nil
	}
	// rest of the code with less indentation
```